docs/README.md

```diff
@@ -100,7 +100,7 @@ Volunteers have translated many of our Contributing, How-to, and Code of Conduct
   - [How-to](HOWTO-sv.md)
 - Tamil / தமிழ்
 - Telugu / తెలుగు
-  - [Code of Conduct](CODE_OF_CONDUCT-te.md)
```

Comment:
Looks like this entry was added from another PR, please add it back

Reply:
ok